### PR TITLE
Associate Hack Camp with Hack Club

### PR DESCRIPTION
**Pull request by KrishMunot**

### CODE REVIEW

**Comment by zachlatta** [comment]
Hmm, seems like Percy invalidated our token, breaking tests. Will have to look into this.

**Comment by zachlatta** [comment]
Got those Percy tests working :raised_hands:

**Comment by maxwofford** [comment]
@zachlatta I'll be on a flight until tomorrow. Do you want to review this?

File: css/index.css [hunk 1]
```diff
@@ -25,6 +27,22 @@ a:hover {
   background-size: cover;
   box-sizing: border-box; }
 
+.hackclub-logo {
+  font-family: "Amaranth", sans-serif;
+}
```
Comment (zachlatta) [inline css/index.css:32]: Is the webfont for Amaranth being loaded on the page? If not, this won't render correctly.

File: index.html [hunk 2]
```diff
@@ -30,9 +30,15 @@
     <meta content='https://camp.hackclub.com/img/hack_camp_group_photo.jpg' name='og:image'>
     <meta content="Hack Camp is a two-week summer program designed for high school students passionate about technology. We'll bring you from your first line of code to publishing your first project." name='og:description'>
   </head>
-  <body>
+  <body>
```
Comment (zachlatta) [inline index.html:33]: Are the extra spaces on this line intentional?

File: index.html [hunk 3]
```diff
@@ -30,9 +30,15 @@
...
-  <body>
+  <body>      
     <div class='row text-center section title'>
       <div class='small-11 medium-9 large-4 small-centered'>
+      <div class="hackclub-logo">
+      <img src="/img/favicon.ico">
+      <h3>
+        <a href="https://hackclub.com">Hack Club's</a><br>
+      </h3>
+      </div>
```
Comment (zachlatta) [inline index.html:41]: Should lines 37 through 40 be indented?

File: css/index.css [hunk 4]
```diff
@@ -14,6 +14,8 @@ a:hover {
   color: #ff1825;
 }
 
+
+
```
Comment (zachlatta) [inline css/index.css:18]: Are these extra newlines intentional?

File: css/index.css (hunk 1 above)
Comment (zachlatta) [inline css/index.css:32]: Perhaps a better approach would be to use Hack Club's SVG logo.

A white version is over at https://github.com/hackclub/meta/blob/07b909a47f432c3b294afd735313025b47ad205a/logos/hack_club_full_white.svg.

File: index.html [hunk 5]
```diff
@@ -42,6 +48,7 @@
           Apply Now
         </a>
       </div>
+
```
Comment (zachlatta) [inline index.html:51]: Is this extra newline intentional?

File: index.html [hunk 6]
```diff
@@ -222,7 +229,8 @@
         <div class='small-12 columns'>
           <h2>Any Questions?</h2>
           <p>
-            Please reach out! We can be reached anytime at <a class="email" href="mailto:summer@hackclub.com">summer@hackclub.com</a>.
+            Please reach out! We can be reached anytime at <a class="email" href="mailto:summer@hackclub.com">summer@hackclub.com</a>.<br>
+             © 2016 <a class="email" href="https://hackclub.com">Hack Club</a>, a 501(c)(3) non-profit [Tax ID46-0945785].
```
Comment (zachlatta) [inline index.html:233]: The tax id section should have a space after `ID`. So it should be `[Tax ID 46-0945785]`.

File: index.html [hunk 7]
```diff
@@ -30,9 +30,15 @@
...
     <meta content="Hack Camp is a two-week summer program designed for high school students passionate about technology. We'll bring you from your first line of code to publishing your first project." name='og:description'>
   </head>
-  <body>
+  <body>      
     <div class='row text-center section title'>
       <div class='small-11 medium-9 large-4 small-centered'>
+      <div class="hackclub-logo">
+      <img src="/img/favicon.ico">
+      <h3>
+        <a href="https://hackclub.com">Hack Club's</a><br>
```
Comment (zachlatta) [inline index.html:39]: Can you make it so this link is only underline on hover?

**Comment by zachlatta** [comment]
Looking really good so far. I left a few minor comments, will merge after they're addressed.

**Comment by maxwofford** [comment]
Going to close this for the time being because it looks like it hasn't been updated in a while. Feel free to reopen!

**Pull request closed without merging**